bindings/python/pydeck/pydeck/io/html.py

```diff
@@ -20,8 +20,7 @@ def render_json_to_html(json_input, mapbox_key=None, tooltip=True):
     html_str = js.render(
         mapbox_key=mapbox_key,
         json_input=json_input,
-        # TODO change before publication to the NPM-hosted module
-        deckgl_jupyter_widget_bundle='',
+        deckgl_jupyter_widget_bundle='https://cdn.jsdelivr.net/npm/@deck.gl/jupyter-widget@8.0.0-alpha.2/dist/index.js',
```

Comment:
Is there a reason to lock the version of the package?

Reply:
Guaranteeing consistent behavior from one pydeck release to the next

Reply:
- There should not be any breaking changes between patch releases of `@deck.gl/jupyter-widget`.
- The pydeck release should require a minimum version to ensure certain bug fix/new feature is included. We frequently publish fixes to the core module and pydeck is not automatically released with the js packages. There is no reason for you to release a new version of pydeck without any change on the python side every time deck.gl is published.

Reply:
Locked to deck.gl ^8.0 in the latest commit